docs/reference/http-api.md

```diff
@@ -363,36 +366,36 @@ By calling get on the *commits* endpoint along with a DocID gives you access to
     }
     ```
 
-### Apply a commit to document
+### Apply a commit to stream
 
 **:octicons-alert-16: Disabled in gateway mode**
 
-In order to modify a document we apply a commit to its docment log. This commit usually contains a signature over a *json-patch* diff describing a modification to the document contents. The commit also needs to contain pointers to the previous commit and other metadata. You can read more about this in the [Ceramic Specification](https://github.com/ceramicnetwork/ceramic/blob/master/SPECIFICATION.md#document-records){:target="_blank"}. Different document types may have different formats for their commits. If you want to see an example implementation for how to construct a commit you can have a look at the implementation of the TileDoctype.
+In order to modify a stream we apply a commit to its log. This commit usually contains a signature over a *json-patch* diff describing a modification to the stream contents. The commit also needs to contain pointers to the previous commit and other metadata. You can read more about this in the [Ceramic Specification](https://github.com/ceramicnetwork/ceramic/blob/master/SPECIFICATION.md#document-records){:target="_blank"}. Different stream types may have different formats for their commits. If you want to see an example implementation for how to construct a commit you can have a look at the implementation of the TileDocument.
 
 === "Request"
-    
+
     ```bash
     POST /api/v0/commits
     ```
 
     #### Request body fields:
 
-    - `docId` - the name of the doctype to use, string
-    - `commit` - the content of the commit to apply (will differ per doctype)
-    - `docOpts` - options for the document update [DocOpts](https://developers.ceramic.network/reference/typescript/interfaces/_ceramicnetwork_common.docopts-1.html){:target="_blank"} (optional)
+    - `streamId` - the StreamID of the stream to apply the commit to, string
+    - `commit` - the content of the commit to apply (will differ per streamtype)
```

Comment:
One thing that's annoying about this api is that it's not possible to post multiple commits in one request.